recipes/fortuno/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "fortuno" %}
+{% set version = "0.1.0" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fortuno-repos/fortuno/archive/v{{ version }}.tar.gz
+  sha256: 9639be1159e03d5e31032af6aa9d4f8483d1f3df0b84a3b43a564273a68ff5e1
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  {% if mpi != "nompi" %}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    # this is necessary for all compiled recipes
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+    - pkg-config
+  host:
+    - {{ mpi }}  # [mpi != "nompi"]
+  run:
+    - {{ mpi }}  # [mpi != "nompi"]
+
+test:
+  source_files:
+    - test/export/**
+  requires:
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - {{ mpi }}  # [mpi != "nompi"]
+    - cmake
+    - meson
+    - ninja
+    - fypp
+    - pkg-config
+  commands:
+    - pkg-config fortuno --exact-version={{ version }}
+    - pkg-config fortuno_serial --exact-version={{ version }}
+    - pkg-config fortuno_mpi --exact-version={{ version }}  # [mpi != "nompi"]
+
+about:
+  home: https://github.com/fortuno-repos/fortuno
+  summary: 'Flextensible unit testing framework for Fortran'
+  description: |
+    Fortuno (Fortran Unit Testing Objects) is a flexible & extensible, object-oriented unit testing
+    framework designed for the Fortran programming language. It emphasizes ease of use by minimizing
+    boiler plate code when writing tests, while also prioratizing modularity and extensibility.
+    Fortuno provides the essential building blocks to help developers create customized unit testing
+    solutions.
+  license: BSD-2-Clause-Patent
+  license_family: BSD
+  # It is required to include a license file in the package,
```

Comment:
Please remove these comments. They don't help the recipe and are well defined in conda-forge's docs.